app/src/main/java/com/a0xffffffff/dinesum/NewRequestFragment.java

```diff
@@ -175,8 +175,10 @@ public void onTimeSet(
         mSubmitButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                String startTime = Integer.toString(mStartHour) + ":" + Integer.toString(mStartMinute);
-                String endTime = Integer.toString(mEndHour) + ":" + Integer.toString(mEndMinute);
+                String pad = mStartMinute < 10 ? "0" : "";
+                String startTime = Integer.toString(mStartHour) + ":" + pad + Integer.toString(mStartMinute);

```

Comment:
do the hours automatically get padded to two digits? so that "9" becomes "09", etc.

Reply:
wait i think the minutes automatically get padded but not the hours, ie. "4:05" gets stored in the db